ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLong.java

```diff
@@ -18,6 +18,11 @@
     private double latitude;
     private double longitude;
 
+    public LatLong(){
```

Comment:
@billbonney I don't agree with having a default constructor. This is usually the cause of subtle and hard to find bugs. In which scenario would having a default constructor be helpful?